mesonbuild/compilers/java.py

```diff
@@ -113,3 +113,11 @@ def sanity_check(self, work_dir, environment):
 
     def needs_static_linker(self):
         return False
+
+    def find_library(self, libname, env, extra_dirs):
+        for d in extra_dirs:
+            trial = os.path.join(d, libname)
+            if os.path.isfile(trial):
+                return ['-cp', trial]
+        return None
+
```

Comment:
__[Flake8]__

[W391] blank line at end of file


<!-- Comment by [SideCI](https://sideci.com) -->